### ISSUE TITLE
HexWeb 1.0.3 causing server crash: java.lang.NoClassDefFoundError: okio/Buffer

### ISSUE THREAD

**Issue by loliking-01** [TechTastic/HexWeb#3]
[crash-2024-12-08_21.05.44-server.txt](https://github.com/user-attachments/files/18055606/crash-2024-12-08_21.05.44-server.txt)

**Comment by TechTastic** [comment]
This may be a Sinytra Connector incompat...

**Comment by loliking-01** [comment]
> This may be a Sinytra Connector incompat...

Can it be compatible with the iota type of other auxiliary modules?
I set up my own server with a database installed to store the list of spells. As a result, because it is not compatible with other types, the practicality is not as good as that of the little yellow duck

**Comment by loliking-01** [comment]
> I dont know what you mean exactly
Your mod has added a new iota type of mod to Hexical. Unable to convert json save

**Comment by loliking-01** [comment]
> I dont know what you mean exactly
I'd love to see you be able to make the 1.20.1 version of the subsidiary module iota type compatible, I personally think the module you made is very good.

**Comment by TechTastic** [comment]
ah i understand you now

you want me to allow my JSON iota to be able to take in Hexical's Pigment, Identifier, and Dye iotas

**Comment by loliking-01** [comment]
> ah i understand you now
> 
> you want me to allow my JSON iota to be able to take in Hexical's Pigment, Identifier, and Dye iotas

Yes, although it is unknown when the authors of moreiotas will re-release version 1.20.1. But I cannot convert the iota type of the mod to JSON format.
Most of the available iota types for my Fabric server are currently on these two mods, even though Moreiotas is not officially released

**Comment by TechTastic** [comment]
Oh, you are trying to parse a StringIota as a JsonIota

its less about MoreIotas and more about how `com.google.gson.JsonParser` actually tries to parse the string

**Comment by TechTastic** [comment]
![Image](https://github.com/user-attachments/assets/7789d918-190c-4cbc-9a98-3d2a21845159)

**Comment by TechTastic** [comment]
this is how the pattern works in the code

**Comment by loliking-01** [comment]
![Image](https://github.com/user-attachments/assets/a1b4e565-70c3-4740-ac45-1d31f67cf56d)

![Image](https://github.com/user-attachments/assets/4d9ed7da-f94e-48c2-8129-066b09c514ed)

![Image](https://github.com/user-attachments/assets/8fbe166f-686f-49aa-bbc5-a4e049ce46df)

![Image](https://github.com/user-attachments/assets/6af116b9-bb38-428b-a132-46a260e2cfad)

I can't parse this iota into JSON, which is why I want you to be compatible.

**Comment by TechTastic** [comment]
1) I wont do entities or entity types as the entity/entity type in question is world/modpack specific

2) I wont do items for the same reason

3) i may support Identifier in the future

**Comment by loliking-01** [comment]
thank you

**Comment by TechTastic** [comment]
1.0.5 now parses all 3

**Comment by loliking-01** [comment]
> 1.0.5 now parses all 3

good
